enterprise/api_client/lms.py

```diff
@@ -309,6 +312,13 @@ class ThirdPartyAuthApiClient(UserAPIClient):
 
     API_BASE_URL = urljoin(f"{settings.LMS_INTERNAL_ROOT_URL}/", "api/third_party_auth/v0/")
 
+    def __init__(self, user, expires_in=settings.OAUTH_ID_TOKEN_EXPIRATION):
+        if get_value:
+            self.API_BASE_URL = urljoin(
+                f"{get_value('LMS_INTERNAL_ROOT_URL', settings.LMS_INTERNAL_ROOT_URL)}/", "api/third_party_auth/v0/",
```

Comment:
Shouldn't be better to get the site from the EnterpriseCustomer.site instead of the site configurations? @alexjmpb

Reply:
I preferred to use a site configuration to avoid affecting other instances of this client. Before this, the base URL was being obtained from the setting LMS_INTERNAL_ROOT_URL with the site configuration we are able to control this value per site while maintaining this setting.